SonarRunner.Shim/RoslynV1SarifFixer.cs

```diff
@@ -46,7 +51,7 @@ public static class RoslynV1SarifFixer
         /// <summary>
         /// The low-level implementation of the the fix - applying escaping to backslashes and quotes.
         /// </summary>
-        private static string ApplyFixToSarif(string unfixedSarif)
+        private string ApplyFixToSarif(string unfixedSarif)
         {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The 'this' parameter (or 'Me' in Visual Basic) of 'RoslynV1SarifFixer.ApplyFixToSarif(string)' is never used. Mark the member as static (or Shared in Visual Basic) or use 'this'/'Me' in the method body or at least one property accessor, if appropriate. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=fxcop%3AMarkMembersAsStatic)